### PR TITLE
Extend and add tests for PR #463

### PR DESCRIPTION
**Pull request by lucatume**
This PR wraps and refactors the orignal #463 PR from @gabe-connoly to use XPath to find the logout element, add tests and update the documentation.

### CODE REVIEW

File: docs/modules/WPBrowser.md [hunk 1]
```diff
@@ -322,16 +344,16 @@ $I->loginAsAdmin();
 
 <hr>
 
-<p>In an administration screen look for an error admin notice. The check is class-based to decouple from internationalization. The method will <strong>not</strong> handle authentication and navigation the administration area.</p>
+<p>In an administration screen look for an error admin notice. The check is class-based to decouple from internationalization. The method will <strong>not</strong> handle authentication and navigation the administration area. <code>.notice.notice-error</code> ones.</p>
 ```php
-  $I->loginAsAdmin();
+$I->loginAsAdmin()ja
```
Comment (Luc45) [inline docs/modules/WPBrowser.md:349]: `ja`?

File: docs/modules/WPBrowser.md [hunk 2]
```diff
@@ -340,14 +362,14 @@ $I->loginAsAdmin();
 
 <p>In an administration screen look for an admin notice. The check is class-based to decouple from internationalization. The method will <strong>not</strong> handle authentication and navigation the administration area.</p>
 ```php
-  $I->loginAsAdmin();
+$I->loginAsAdmin()ja
```
Comment (Luc45) [inline docs/modules/WPBrowser.md:365]: `ja`?

Comment (lucatume) [reply docs/modules/WPBrowser.md:365]: You do not understand art.

File: docs/modules/WPBrowser.md (hunk 1 above)
Comment (lucatume) [reply docs/modules/WPBrowser.md:349]: More art you do not understand.

**Pull request merged**